packages/graph-explorer/src/modules/EntityDetails/EdgeDetail.tsx

```diff
@@ -37,7 +37,7 @@ const EdgeDetail = ({ edge }: EdgeDetailProps) => {
         </div>
         <div className={"content"}>
           <div className={"title"}>{edge.displayTypes}</div>
-          {edge.displayId && <div>{edge.displayId}</div>}
+          {edge.hasUniqueId ? <div>{edge.displayId}</div> : null}
```

Comment:
Curious why this check is preferred to the original check for `displayId`? Is there still value in setting the `displayId` on the `EdgeDetail` for SPARQL if it's not shown?

Reply:
I honestly don't remember why I made that change. If I had to guess it was to make sure no properties were optional on the model. But that's not a good enough reason.

I reverted the change.

Reply:
Actually, I take it all back. After reverting the change and seeing some errors I now remember why. It is about optionality, but specifically around the logic to get the "display name" value. Since `displayId` is an option on non-sparql databases if it was optional it would require display name to be optional, which I do not want.